projects/uitdatabank/toc.json

```diff
@@ -135,26 +256,44 @@
         }
       ]
     },
+    {
+      "type": "item",
+      "title": "API reference",
+      "uri": "reference/search.json"
+    },
     {
       "type": "divider",
       "title": "Taxonomy API"
     },
     {
       "type": "item",
-      "title": "API reference",
-      "uri": "reference/taxonomy.json"
+      "title": "Introduction",
+      "uri": "docs/taxonomy-api/introduction.md",
+      "slug": "taxonomy-api/introduction"
+    },
+    {
+      "type": "item",
+      "title": "Authentication",
+      "uri": "docs/taxonomy-api/authentication.md",
+      "slug": "taxonomy-api/authentication"
     },
     {
       "type": "group",
       "title": "Guides",
       "items": [
         {
           "type": "item",
-          "title": "Terms",
-          "uri": "docs/terms.md"
+          "title": "What are terms?",
+          "uri": "docs/taxonomy-api/terms.md",
+          "slug": "taxonomy-api/what-are-terms"
```

Comment:
I will provide a redirect for the current URL to the new slug (because the file was moved as well so the current URL won't work anymore, contrary to when we only add a slug)